pgml-extension/src/bindings/chunking.py

```diff
@@ -0,0 +1,29 @@
+from langchain.text_splitter import (
```

Comment:
I'd rename this to langchain, since other bindings refer to the library dependency rather than the functionality.

Reply:
Yup